### PR TITLE
feat(changelog): public-gateways-changed-the-public-gateway-is 2023-07-18

### PR DESCRIPTION
**Pull request by ofranc**
Reporter: Benedikt Rollik

This PR is a new changelog contribution. It has been created automatically.

Make sure to review it carefully before merging it.

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: changelog/july2023/2023-07-18-public-gateways-changed-the-public-gateway-is.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: The Public Gateway is changing
```
Comment (nerda-codes) [inline changelog/july2023/2023-07-18-public-gateways-changed-the-public-gateway-is.mdx:2]:
```suggestion
title: Public Gateway is changing
```

File: changelog/july2023/2023-07-18-public-gateways-changed-the-public-gateway-is.mdx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+status: changed
+author:
+    fullname: 'Join the #public-gateway channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2023-07-18
+category: network
+product: public-gateways
+---
+
+With the release of VPC in GA, the Public Gateway is no longer supporting the DHCP server.
```
Comment (nerda-codes) [inline changelog/july2023/2023-07-18-public-gateways-changed-the-public-gateway-is.mdx:12]:
```suggestion
With the release of VPC in GA, Public Gateway no longer supports the DHCP server.
```

File: changelog/july2023/2023-07-18-public-gateways-changed-the-public-gateway-is.mdx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+The Public Gateway focuses now on its primary function such as
```
Comment (nerda-codes) [inline changelog/july2023/2023-07-18-public-gateways-changed-the-public-gateway-is.mdx:13]:
```suggestion
Public Gateway now focuses on its primary functions such as:
```

File: changelog/july2023/2023-07-18-public-gateways-changed-the-public-gateway-is.mdx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+- Internet access: Allowing external traffic to access the internal resources of a Private Network
```
Comment (nerda-codes) [inline changelog/july2023/2023-07-18-public-gateways-changed-the-public-gateway-is.mdx:14]:
```suggestion
- Internet access: allowing external traffic to access the internal resources of a Private Network
```

File: changelog/july2023/2023-07-18-public-gateways-changed-the-public-gateway-is.mdx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+- NAT-Gateway: Distributing request to the right IP address
```
Comment (nerda-codes) [inline changelog/july2023/2023-07-18-public-gateways-changed-the-public-gateway-is.mdx:15]:
```suggestion
- NAT-Gateway: distributing requests to the right IP addresses
```

File: changelog/july2023/2023-07-18-public-gateways-changed-the-public-gateway-is.mdx [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+- SSH Bastion: Allowing and monitoring SSH connection
```
Comment (nerda-codes) [inline changelog/july2023/2023-07-18-public-gateways-changed-the-public-gateway-is.mdx:16]:
```suggestion
- SSH Bastion: allowing and monitoring SSH connections
```

File: changelog/july2023/2023-07-18-public-gateways-changed-the-public-gateway-is.mdx [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+category: network
+product: public-gateways
+---
+
+With the release of VPC in GA, Public Gateway no longer supports the DHCP server.
+Public Gateway now focuses on its primary functions such as:
+- Internet access: allowing external traffic to access the internal resources of a Private Network
+- NAT-Gateway: distributing requests to the right IP addresses
+- SSH Bastion: allowing and monitoring SSH connections
+- Local DNS server: customizing resources name on the Private Network
```
Comment (RoRoJ) [inline changelog/july2023/2023-07-18-public-gateways-changed-the-public-gateway-is.mdx:17]:
```suggestion
- Local DNS server: customizing resource names on the Private Network
```

File: changelog/july2023/2023-07-18-public-gateways-changed-the-public-gateway-is.mdx [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+
```
Comment (RoRoJ) [inline changelog/july2023/2023-07-18-public-gateways-changed-the-public-gateway-is.mdx:18]: See our [dedicated documentation](https://www.scaleway.com/en/docs/network/vpc/reference-content/vpc-migration/) for full details.

**Pull request merged**